### ISSUE TITLE
Outlook Mac Thick Client : Validated Attachments in the Outlook Web Addin popup are not getting downloaded . 

### ISSUE THREAD

**Issue by soumyamondal-accenture** [OfficeDev/office-js#3824]
Outlook Mac Thick Client : Validated Attachments in the Outlook Web Addin popup are not getting downloaded . #3662

creating a new ticket as a follow up  to :-https://github.com/OfficeDev/office-js/issues/3662, as the the solution is yet to be found.

**Comment by exextoc** [comment]
@soumyamondal-accenture Did you try the workaround suggested in issue #3662? 
Adding here again. Could you please try the workaround suggested here https://github.com/OfficeDev/office-js/issues/3#issuecomment-688251989 and see if it resolves the issue?
Also as mentioned here https://github.com/OfficeDev/office-js/issues/3#issuecomment-641041099, we are blocked by a [WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=174076). When we hear back on the WebKit bug, we will re-activate as needed.

**Comment by mailchecktool-admin** [comment]
https://github.com/OfficeDev/office-js/issues/3#issuecomment-558532628
I am facing the same problem as here. Called Office.context.ui.openBrowserWindow(URL); as a solution to the above problem,
I tried to open the browser using the URL I own but it does not work.
Specifically,
After creating the BLOB object
const fileURL = window.URL.createObjectURL(blob)
window.open(fileURL)
was changed to
After creating the BLOB object
const fileURL = window.URL.createObjectURL(blob)
Office.context.ui.openBrowserWindow(fileURL)
but Office.context.ui.openBrowserWindow() could not open the BLOB object URL.
Can't Office.context.ui.openBrowserWindow() open BLOB object URLs directly?

**Comment by exextoc** [comment]
@mailchecktool-admin Please create a separate bug for your issue.

**Comment by soumyamondal-accenture** [comment]
@exextoc  i work with the author of https://github.com/OfficeDev/office-js/issues/3662 in the same team.
Can you please refer to her last comments and provide some insights.

**Comment by exextoc** [comment]
Closing this issue as it is a duplicate of #3662. Please track the same for updates.
